x-pack/plugins/ml/server/routes/annotations.ts

```diff
@@ -86,6 +89,9 @@ export function annotationRoutes(
       validate: {
         body: indexAnnotationSchema,
       },
+      options: {
+        tags: ['access:ml:canCreateJob'],
```

Comment:
I think a user should be allowed to create an annotation, even if they can't create jobs. Maybe this should be `access:ml:canGetJobs` or we could create new capability(s) for adding / deleting annotations?